### PR TITLE
Extract permitted params lists

### PR DESCRIPTION
**Pull request by matiasgarciaisaia**
So they can be easily extended by (...wait for it...) _extensions_.

### CODE REVIEW

File: pages/app/controllers/refinery/admin/page_parts_controller.rb [hunk 1]
```diff
@@ -23,9 +23,12 @@ def destroy
 
       protected
         def new_page_part_params
-          params.except(:part_index).permit(:title, :slug, :body, :locale)
+          params.except(:part_index).permit(permitted_new_page_part_params)
         end
 
+        def permitted_new_page_part_params
```
Comment (bricesanchez) [inline pages/app/controllers/refinery/admin/page_parts_controller.rb]: This method should be `private` like all other `permitted` methods in Refinery

**Comment by matiasgarciaisaia** [comment]
👌 Fixed that! I can squash/rebase the commits before merging - made a different commit to help with reviewing.

**Comment by bricesanchez** [comment]
@matiasgarciaisaia it will be perfect if you could do this, it's better for tracking changes in the git log! Thank you very much!

**Comment by matiasgarciaisaia** [comment]
There! ☝️

**Comment by bricesanchez** [comment]
Thank you @matiasgarciaisaia !

**Pull request merged**